spec/factories/users.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :user do
+    email { "test@email.com" }
```

Comment:
Why you hardcode this ones? Please define rspec helpers with `faker` gem under the hood.

Reply:
Replaced with Faker::Name and ::Email

Reply:
Please move it to custom helpers like `random_email`, `random_password`

Reply:
Done